docs/api-docs/storefront/widgets/widget-builder.md

```diff
@@ -0,0 +1,187 @@
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme: info -->
+
+> Refer to the [Widget Builder README.md](https://github.com/bigcommerce/widget-builder) for the latest supported `node` version.
+
+</div>
+</div>
+</div>
+
+## Install Widget Builder
+
+Follow these instructions to install Widget Builder and its dependencies. 
+
+1. In a terminal, clone the [widget-builder](https://github.com/bigcommerce/widget-builder) GitHub repo.
+
+2. Navigate to the `widget-builder` directory and install Widget Builder.
+
+```shell
+npm run install-cli
+```
+
+3. To retrieve usage information and view supported options and commands, run the following command:
+
+```shell
+widget-builder -h
+```
+
+**Sample output:**
+
+```shell
+Usage: widget-builder [options] [command]
+
+Options:
+  -V, --version                  output the version number
+  -h, --help                     display help for command
+
+Commands:
+  init                           Initialization of widget builder configuration
+  start [options] [widgetPath]   starts the widget builder locally
+  validate [options] <file>
+  create <widget-template-name>  Create a blank widget template
+  publish <widget-template>      Releases the widget template to the store
+                                 belonging to the env config
+  help [command]                 display help for command
+```
+
+## Configure Widget Builder
+
+Once you have installed Widget Builder, the next step is to set up the widget development environment for your store.  This will authorize you to access your store's data during the development process. You will need the store's `Client ID`, `Access Token`, and `API Path` to generate the appropriate configuration. 
+
+To get started, locate or create a parent directory for the widgets you plan to develop using these credentials.
+
+**Sample input:**
+
+```shell
+# organized by name of store  
+cd ~/bc-projects/tools-r-us
+mkdir widget-workshop
+cd widget-workshop
+```
+
+Next, start the interactive sequence that will generate a `.env` file to authenticate the widgets you build in the subfolders of this parent directory.  The session will prompt you to enter your API account credentials. Begin by running `widget-builder init`. 
+
+**Sample session:**
+
+```shell
+~/bc-projects/tools-r-us/widget-workshop % widget-builder init
+
+Thank you for using Widget Builder
+
+            
+This guide will help you get your environment set up.
+
+Before continuing, please make sure you've created or received a Store API account.
+You'll need those credentials in order to generate the appropriate configurations.
+You can find more information here. https://support.bigcommerce.com/s/article/Store-API-Accounts#creating
+
+? Are you ready to continue? You may press any key to continue (Y/n) Yes
+? What is the Client ID? xxxxxxxxxxxxx
+? What is the Access Token? xxxxxxxxxxxxx
+? What is the API Path? https://api.bigcommerce.com/stores/xxxxx/v3/
```

Comment:
? What is the API Path? https://api.bigcommerce.com/stores/{{store-hash}}/v3/

Reply:
more or less.  It's in the file you download when you generate the API access token.  I will add a bit on line `26` to clarify.

Reply:
@traci707 This is part of the shell text, so not much to be done about that

Reply:
@markcmurphy it's a representation of an interactive session, so it's a good place to future-proof & indicate that these values come from the .txt you generate in the API Accounts section of the dashboard.

Reply:
see most recent.